addons/account_peppol/wizard/account_move_send_views.xml

```diff
@@ -9,10 +9,13 @@
                 <xpath expr="//div[@name='warnings']" position="inside">
                     <field name="enable_peppol" invisible="1"/>
                     <div class="alert alert-warning" role="alert" attrs="{
-                                'invisible': ['|',
+                                'invisible': ['|', '|',
                                     ('checkbox_send_peppol', '!=', True),
                                     ('peppol_warning', '=', False),
+                                    ('ubl_partner_warning', '!=', False),
```

Comment:
You break the stable policy here. If you update account_peppol without account_edi_ubl_cii with your new field, the view will crash because `ubl_partner_warning` is not inside the view :(